docs/src/components/Section.js

```diff
@@ -0,0 +1,47 @@
+// @flow strict
+import React, { useEffect, useState } from 'react';
+import { Box, TapArea, Text } from 'gestalt';
+import { useLocation } from 'react-router-dom';
+import Collapsable from './Collapsable.js';
+
+export default function Section({
+  section,
+  groups,
+}: {|
+  section: string,
+  groups: { [string]: Array<string> },
```

Comment:
@AlbertCarreras please use exact object syntax in here - `groups: {| [string]: Array<string> |}`

Reply:
done